README.md

```diff
@@ -217,6 +217,18 @@ The ViewModifiers are used to customise the look and feel of the BottomSheet.
 - Changing the threshold does not affect whether either option is enabled.
 - The default threshold is 30% (0.3).
 
+`.iPadFloatingSheet(Bool)`: Makes the iPad's bottom sheet be an actual bottom sheet when set to true.
+- The default is `false`.
+
+`.sheetWidth(BottomSheetWidth)`: Sets the width of the sheet.
+- Can be relative through `BottomSheetWidth.relative(x)`.
+- Can be absolute through `BottomSheetWidth.absolute(x)`.
+- Set to `BottomSheetWidth.platformDefault` to let the library decide the width.
+
+`.accountForKeyboardHeight(Bool)`: Adds bottom padding to the main content equal to the keyboard height when the keyboard appears.
+- The default is `false`.
```

Comment:
I normally don't mention default values on Boolean values. Because all modifiers are meant to change some default behavior. But maybe I should change that; what is your opinion? (Same above)